scripts/just.config.ts

```diff
@@ -99,8 +99,26 @@ export function preset() {
   task('storybook:start', startStorybookTask());
   task('storybook:build', buildStorybookTask());
 
-  task('fluentui:publish:patch', fluentuiLernaPublish('patch'));
-  task('fluentui:publish:minor', fluentuiLernaPublish('minor'));
+  task('fluentui:publish:minor', () => {
```

Comment:
I'd prefer that this not be a just task, at least not in the main just config. It adds extra overhead to every build step of every task. Better to either make a separate just config for scripts (we need to do that anyway) or just run a script file directly (can be written in TS if you run it like `node -r ./scripts/ts-node-register ./scripts/whatever.ts`)

Reply:
I moved all scripts regarding publishing process of fluent ui into `scripts/fluentui-publish` and run it from node :)